Automattic/wp-calypso
# Jetpack New Pricing Page - Fix legacy products shown in featured section

#### Proposed Changes

* If a user owns a legacy product (Daily or Realtime) it is shown in the featured section, made fix to show only T1 and T2 plans
* Same with Security Bundles. Show only T1 and T2 bundles in the featured section

#### Testing Instructions
* Launch a [JN Site](https://jurassic.ninja/specialops/) with legacy plans (Jetpack Daily or Realtime) if you don't have any site with one already
* click on Jetpack Cloud live link below
    * Goto `/pricing?flags=jetpack/pricing-page-rework-v1`
* or boot up this PR 
    * Run `git fetch && git checkout fix/featured-section-legacy-products`
    * Run `yarn start-jetpack-cloud`
    * Goto http://jetpack.cloud.localhost:3000/pricing/:siteId (replace siteId with your JN site)
* Make sure no Legacy plans are shown in the **Most popular products** section and confirm it is shown in the **All Products** section
* Repeat the same step for Bundles by launching a JN Site with legacy Bundles (Jetpack Security Daily or Jetpack Security Realtime) and make sure it is not shown in the **Most popular bundles** section in Bundles Tab

#### Screenshot 

##### Before
<img width="1526" alt="Screenshot 2022-10-12 at 8 29 05 AM" src="https://user-images.githubusercontent.com/2027003/195239817-90c020a5-ba39-43e2-8992-cd953a1ff9ed.png">


##### After
<img width="1295" alt="Screenshot 2022-10-12 at 8 29 46 AM" src="https://user-images.githubusercontent.com/2027003/195239835-da40053e-0143-460d-acc8-7927ce37033b.png">


#### Pre-merge Checklist






Related to #
